e2e/nomostest/clean.go

```diff
@@ -244,42 +171,58 @@ func DeleteManagedNamespaces(nt *NT) error {
 		return err
 	}
 
-	// Error if any protected namespace are managed by config sync
-	// (managed-by label added) and not reverted by the test.
+	// Error if any protected namespaces are managed by config sync.
+	// These must be cleaned up by the test.
 	protectedNamespacesWithTestLabel, nsListItems := filterNamespaces(nsList.Items,
 		protectedNamespaces...)
 	if len(protectedNamespacesWithTestLabel) > 0 {
-		return errors.Errorf("protected namespace(s) managed by config sync and cannot be cleaned: %+v",
+		return errors.Errorf("protected namespace(s) still have config sync labels and cannot be cleaned: %+v",
+			protectedNamespacesWithTestLabel)
+	}
+	return deleteNamespaces(nt, nsListItems)
+}
+
+// deleteImplicitNamespaces deletes the namespaces with the PreventDeletion annotation.
+func deleteImplicitNamespaces(nt *NT) error {
```

Comment:
nit: looks like the order of this function was changed which makes the diff larger than it needs to be

Reply:
Yeah, I moved it up with the other delete and delete namespace functions.

Reply:
Perhaps that was snarky. Sorry. What I meant was: I agree with you but I don't want to do it right now. If you feel inspired, feel free.